### PR TITLE
Clean up in-line f-string concatenation

### PR DESCRIPTION
**Pull request by josh-fell**
A general clean up of f-strings that implicitly use string literal concatenation when not needed.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragement file, named `{pr_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Comment by potiuk** [comment]
Oh yeah. I wish one of the million automated tools of our would do that automatically ... maybe we need to enable it simply.

**Comment by potiuk** [comment]
Seems we can use https://pypi.org/project/flake8-implicit-str-concat/ to at least prevent it.

**Comment by potiuk** [comment]
Would you want to add flake8-implicit-str-concat to devel and see if it helps @josh-fell ?

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
Fine to merge it :). I just restarted randomly failed jobs, so feel free to merge.

File: airflow/providers/databricks/operators/databricks_repos.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ def __init__(
             self.git_provider = self.__detect_repo_provider__(git_url)
             if self.git_provider is None:
                 raise AirflowException(
-                    "git_provider isn't specified and couldn't be guessed" f" for URL {git_url}"
+                    "git_provider isn't specified and couldn't be guessed for URL {git_url}"
```
Comment (alexott) [inline airflow/providers/databricks/operators/databricks_repos.py:93]: `f` at beginning is missing

Comment (potiuk) [reply airflow/providers/databricks/operators/databricks_repos.py:93]: ups

Comment (alexott) [reply airflow/providers/databricks/operators/databricks_repos.py:93]: I'll fix it in the next PR, it's not critical

Comment (josh-fell) [reply airflow/providers/databricks/operators/databricks_repos.py:93]: Nice catch @alexott!

**Pull request merged**